Update User Roles on Admin Panel

stellazarei opened the pull request:
Adds a new tab to admin panel for changing user roles.

closes #77

bettesworthjayna reviewed (approved):
I can't seem to scroll down the table to the submit button when there are lots of users in the table. Could you encompass the table in a scroll-area or something similar?

Other then that it looks good!

stellazarei commented:
<img width="1023" height="603" alt="Screenshot 2025-07-27 at 6 13 42 PM" src="https://github.com/user-attachments/assets/d6113aa7-847f-44e6-8a70-88b7f8d1d96c" />

Fixed dark mode coloring

Added `overflow-auto` globally. It looks like the scrolling would have also been an issue with other admin tabs.
